vmm/src/vm.rs

```diff
@@ -2658,6 +2658,14 @@ impl Transportable for Vm {
 }
 
 impl Migratable for Vm {
+    fn set_migration_started(&mut self) {
+        self.device_manager.lock().unwrap().set_migration_started();
+    }
+
+    fn clear_migration_started(&mut self) {
```

Comment:
Could you use the `complete_migration` func to handle this?

Reply:
Yes, handle it in complete_migration is more appropriate.